jetbrains-rider/ReSharper.AWS/src/AWS.Daemon/Lambda/LambdaFinder.cs

```diff
@@ -0,0 +1,336 @@
+using System.Collections.Generic;
+using System.Linq;
+using JetBrains.Annotations;
+using JetBrains.Metadata.Reader.API;
+using JetBrains.Metadata.Reader.Impl;
+using JetBrains.ProjectModel;
+using JetBrains.ProjectModel.ProjectsHost.Dependencies;
+using JetBrains.ReSharper.Psi;
+using JetBrains.ReSharper.Psi.CSharp;
+using JetBrains.ReSharper.Psi.Resolve;
+using JetBrains.ReSharper.Psi.Util;
+using JetBrains.UI.ThemedIcons;
+using JetBrains.Util;
+using JetBrains.Util.Logging;
+
+namespace ReSharper.AWS.Lambda
```

Comment:
What tests covers this logic?

Reply:
This is a pease of refactoring for defining lambda handlers. This logic is used LambdaGutterMarkHighlighting and is covered by LambdaGutterMarkHighlightingTest. I left the duplicate code in the original commit. Removed it + removed the unused method to  collect all handlers from backend logic to not mix it for now. Sorry for confusion. Pushed an updated.

Reply:
Okay, sounds good.